main/train_classifier_g.py

```diff
@@ -138,34 +111,23 @@ def predict_unlabeled(unlabeled_data, trained_model, features, G_data=True):
 if __name__ == "__main__":
   start_time = time.time()
   if len(sys.argv) > 1:
-    if sys.argv[1] == "pretrain":
-      logging.info("Training on GH data.")
-      [training, _] = receive_data.receive_data()
-      train_model(training, pretrain=True)
-    if sys.argv[1] == "test":
-      logging.info("Applying pre-trained GH model.")
-      if len(sys.argv) >= 2:
-        # set model
-        model_file = open(sys.argv[2], "rb")
-        trained_model = pickle.load(model_file)
-        model_file.close()
-        [_, unlabeled] = receive_data.receive_data()
-        predict_unlabeled(unlabeled, trained_model)
-      else:
-        logging.info("Please povide the name of the model's pickle file.")
-        logging.info(
-            "The file should be among the data dependencies in the BUILD file.")
+    what_data = sys.argv[1]
+    print(what_data)
```

Comment:
here and elsewhere, is there a reason to print rather than log?

Reply:
I printed this because I wanted to see this information right away. I'll remove it because it was purely for debugging purposes.

Reply:
removed